.travis.yml

```diff
@@ -9,6 +9,8 @@ addons:
         - odbc-postgresql
         - slapd
         - ldap-utils
+    hosts:
+        - muc.localhost
```

Comment:
it adds host entry for 127.0.0.1 into /etc/hosts.
because some tests were failing - mongoose tried to resolve the address and it was failing to resolve.